src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -301,8 +301,8 @@ csm::EcefLocus UsgsAstroFrameSensorModel::imageToRemoteImagingLocus(const csm::I
                                                              csm::WarningList *warnings) const {
   // Find the line,sample on the focal plane (mm)
   // CSM center = 0.5, MDIS IK center = 1.0
-  double col = imagePt.samp - (m_ccdCenter[0] - 0.5);
-  double row = imagePt.line - (m_ccdCenter[1] - 0.5);
+  double col = imagePt.samp - (m_ccdCenter[0]);
+  double row = imagePt.line - (m_ccdCenter[1]);
```

Comment:
I could have sworn this was already merged by #91 weird

Reply:
I bet that upstream was not pulled and merged, so we are seeing both.